.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       matrix:
         rust:
           - stable
-          - 1.71.0
+          - 1.72.0
```

Comment:
Irritatingly, this actually has nothing to do with us, but because the `sha1` crate bumped their MSRV in a patch version.